beeline/src/java/org/apache/hive/beeline/BeeLine.java

```diff
@@ -162,6 +163,9 @@
   public static final String BEELINE_DEFAULT_JDBC_DRIVER = "org.apache.hive.jdbc.HiveDriver";
   public static final String DEFAULT_DATABASE_NAME = "default";
 
+  private static final String BEELINE_SESSION_HOOK_CLASS = "beeline.session.hook.class";
```

Comment:
change to 
BEELINE_SESSION_HOOK_CLASS_VARNAME

Reply:
done.